metadata-ingestion/setup.py

```diff
@@ -51,7 +51,8 @@ def get_long_description():
     # Markupsafe breaking change broke Jinja and some other libs
     # Pinning it to a version which works even though we are not using explicitly
```

Comment:
probably worth removing these comments, but I thought I'd get some feedback on this change first.

Reply:
Doesn't the new constraint allow for versions newer than 2.0.1 ? The original comment here seems to indicate that versions newer than 2.0.1 will break Jinja and other libs.

Reply:
Yea, let me go ahead and just pin it to 2.0.1; The reason I swapped it to what Airflow uses is because I believe the markupsafe breaking change issue is resolved on 2.0.1 and has been pushed off to 2.1.0, based on https://github.com/pallets/markupsafe/issues/286 and https://github.com/pallets/markupsafe/issues/283.  Either way, I'll keep it pinned at 2.0.1.  Thanks!